docs/tempo/website/setup/set-up-test-app.md

```diff
@@ -0,0 +1,167 @@
+                name: default
+                receivers:
+                    jaeger:
+                        protocols:
+                            grpc: null
+                            thrift_binary: null
+                            thrift_compact: null
+                            thrift_http: null
+                    opencensus: null
+                    otlp:
+                        protocols:
+                            grpc: null
+                            http: null
+                    zipkin: null
+                remote_write:
+                  - endpoint: <tempoDistributorServiceEndpoint>
+                    insecure: true  # only add this if TLS is not required
+                scrape_configs:
+                  - bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+                    job_name: kubernetes-pods
+                    kubernetes_sd_configs:
+                      - role: pod
+                    relabel_configs:
+                      - action: replace
+                        source_labels:
+                          - __meta_kubernetes_namespace
+                        target_label: namespace
+                      - action: replace
+                        source_labels:
+                          - __meta_kubernetes_pod_name
+                        target_label: pod
+                      - action: replace
+                        source_labels:
+                          - __meta_kubernetes_pod_container_name
+                        target_label: container
+                    tls_config:
+                        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                        insecure_skip_verify: false
+    ```
+
+    If you have followed the [Tanka Tempo installation example]({{< relref "../setup/tanka" >}}), then the `endpoint` value would be:
+
+    ```bash
+    distributor.tempo.svc.cluster.local:4317
+    ```
+
+1. Apply the ConfigMap with:
+
+    ```bash
+    kubectl apply --namespace default -f agent.yaml
+    ```
+
+1. Deploy Grafana Agent using the procedures from the relevant instructions above.
+
+## Create a Grafana Tempo data source
+
+To allow Grafana to read traces from Tempo, you must create a Tempo data source.
+
+1. Navigate to **Configuration ≫ Data Sources**.
+
+1. Click on **Add data source**.
+
+1. Select **Tempo**.
+
+1. Set the URL to `http://<TEMPO-HOST>:<HTTP-LISTEN-PORT>/`, filling in the path to your gateway and the configured HTTP API prefix. If you have followed the [Tanka Tempo installation example]({{< relref "../setup/tanka.md" >}}), this will be: `http://query-frontend.tempo.svc.cluster.local:3200/`
+
+1. Click **Save & Test**.
+
+You should see a message that says `Data source is working`.
+
+If you see an error that says `Data source is not working: failed to get trace with id: 0`, check your Grafana version.
+
+If your Grafana version is < 8.2.2, this is a bug in Grafana itself and the error can be ignored.
```

Comment:
We mention the required version of 9.0 above, should we drop this note?

Reply:
Yeah, I think that's a great idea. cc @knylander-grafana

Reply:
I'm removing the note about the error message being related to Grafana version 8.2 or older. Instead, I've added a link to upgrading Grafana to 9.0 (since you know someone will try to use the procedure with a system that doesn't quite match the minimum requirements).